src/internal/kopia/backup_bases.go

```diff
@@ -46,35 +53,49 @@ type backupBases struct {
 	disableMergeBases bool
 }
 
-func (bb *backupBases) RemoveMergeBaseByManifestID(manifestID manifest.ID) {
+func (bb *backupBases) SnapshotAssistBases() []ManifestEntry {
+	if bb.disableAssistBases {
+		return nil
+	}
+
+	// Need to use the actual variables here because the functions will return nil
+	// depending on what's been marked as disabled.
+	return append(slices.Clone(bb.assistBases), bb.mergeBases...)
```

Comment:
eventually yes we'll stop reporting merge bases as assist bases. I broke things up here so that I could lay some of the groundwork prior to doing that split. Mostly this is so the PRs stay a bit more manageable since splitting things up requires updates to tests which makes it more difficult to focus on the actual logic changes

Reply:
Thanks, this makes sense!